nl_server/config.py

```diff
@@ -87,43 +88,42 @@ class VertexAIIndexInfo(IndexInfo):
 
 # Defines one embeddings index config.
 @dataclass
-class EmbeddingsInfo:
-  indexes: Dict[str, IndexInfo]
-  models: Dict[str, ModelInfo]
+class EmbeddingsConfig:
+  indexes: Dict[str, IndexConfig]
+  models: Dict[str, ModelConfig]
 
 
 #
 # Parse the input `embeddings.yaml` dict representation into EmbeddingsInfo
 # object.
 #
-def parse(embeddings_map: Dict[str, Dict[str, str]]) -> EmbeddingsInfo:
+def parse(embeddings_map: Dict[str, any]) -> EmbeddingsConfig:
   if embeddings_map['version'] == 1:
     return parse_v1(embeddings_map)
   else:
-    logging.warning('Could not parse embeddings map: unsupported version.')
-    return None
+    raise AssertionError('Could not parse embeddings map: unsupported version.')
 
 
 #
 # Parses the v1 version of the `embeddings.yaml` dict representation into
 # EmbeddingsInfo object.
 #
-def parse_v1(embeddings_map: Dict[str, Dict[str, str]]) -> EmbeddingsInfo:
+def parse_v1(embeddings_map: Dict[str, any]) -> EmbeddingsConfig:
   # parse the models
   models = {}
   for model_name, model_info in embeddings_map.get('models', {}).items():
     model_type = model_info['type']
     if model_type == ModelType.LOCAL:
-      models[model_name] = LocalModelInfo(type=model_type,
-                                          gcs_folder=model_info['gcs_folder'])
+      models[model_name] = LocalModelConfig(type=model_type,
+                                            gcs_folder=model_info['gcs_folder'])
```

Comment:
nit:  space

Reply:
hm, I ran lint and the spacing seems correct

Reply:
ok, nm, I saw an extra space that gh diff highlights